core/storage/user/gae_models.py

```diff
@@ -104,6 +104,18 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.DELETE
 
+    @classmethod
+    def has_reference_to_user_id(cls, user_id):
+        """Check that UserSettingsModel exists for user.
```

Comment:
Here and elsewhere: "that" --> "whether".

("Check that" can mean "Verify that", i.e. you're expecting the answer to be true. "Whether" is better for something that could be true, or could be false.)